src/software/thunderscope/robot_communication.py

```diff
@@ -336,9 +338,13 @@ def __enter__(self) -> "self":
     #   - Must retrieve the time_sent field of the proto,
     #     this means taking the diff of the current time and the omit_thunderloop_processing_time_sent
     #   - Must update the visualized widget component
-    # def __receive_robot_status(self, robot_status: Message) -> None:
-    #     print((time.time() - robot_status.omit_thunderloop_processing_time_sent.epoch_timestamp_seconds) * 1000)
-    #     self.__forward_to_proto_unix_io(RobotStatus, robot_status)
+    #   - Create new proto here, log its time here
+    def __receive_robot_status(self, robot_status: Message) -> None:
+        self.__forward_to_proto_unix_io(
+            RoundTripTime,
+            RoundTripTime(thunderscope_time_received_seconds=time.time())
```

Comment:
Don't forget to update this to be similar to your commented out code:
```suggestion
            RoundTripTime(thunderscope_time_received_seconds=time.time() - robot_status.omit_thunderloop_processing_time_sent.epoch_timestamp_seconds)
```

Reply:
Or I guess you're doing this at the UI. It's just a bit misleading since the protobuf type is called `RoundTripTime`. I think we could update the types name to be related to Robot Statistics or something such that we could also extend in the future with more values.